### DISCUSSION TITLE
How to add text labels to complex graphics

### DISCUSSION THREAD

**Discussion by Kiss-World** [observablehq/plot#2108] [Q&A]
I've realized that my capability with text labels in plotting is quite weak. When it comes to complex charts, I don't know how to proceed, and there are not many examples provided officially. For instance, in the scenario described for the chart, I'm unsure how to handle the labels. Are there any more examples of complex charts that I can study to learn about text labels handling, especially in cases involving statistics?
https://observablehq.com/@observablehq/plot-difference-arrows?intent=fork
![image](https://github.com/user-attachments/assets/e42ae19c-e6c4-4761-b9c4-da23bf090966)

**Comment by mythmon** [comment]
Can you be more specific about what kind of text handling and label handling you'd like to learn how to do?
